src/common/constants.ts

```diff
@@ -92,7 +92,7 @@ export const DEFAULT_NETWORK_LIST = [
   },
 ];
 
-export const FATHOM_ID = NEXT_PUBLIC_FATHOM_ID || process.env.NEXT_PUBLIC_FATHOM_ID;
+export const FATHOM_ID = 'YAKFSIOZ' || NEXT_PUBLIC_FATHOM_ID || process.env.NEXT_PUBLIC_FATHOM_ID;
```

Comment:
Do we want to be hardcoding this? None of the rest of this expression will ever be reached.